com.unity.ml-agents/Runtime/Sensor/RayPerceptionSensorComponentBase.cs

```diff
@@ -4,69 +4,123 @@
 
 namespace MLAgents
 {
+    /// <summary>
```

Comment:
Sorry, this file is going to have a lot of conflicts with my other PR.

Reply:
No worries.